test/spreadProp.js

```diff
@@ -0,0 +1,34 @@
+import RA from '../src/index';
+import eq from './shared/eq';
+
+
+describe('spreadProp', function() {
```

Comment:
Redo these tests according what I explained on `spreadPath` tests and I think we should have more tests here

 - [x] currying
 - [ ] non existing prop
 - [x] prop that is non object
 - [ ] test for overriding existing props on provided object
 - [ ] test for standard usecase